firefox_puppeteer/ui/windows.py

```diff
@@ -293,14 +300,15 @@ def get_property(self, property_id):
         """
         return self._l10n.get_property(self.properties, property_id)
 
-    def open_window(self, callback=None, expected_window_class=None):
+    def open_window(self, callback=None, expected_window_class=None, is_focused=True):
         """Opens a new top-level chrome window.
 
         :param callback: Optional, function to trigger the window to open. It is
          triggered with the current :class:`BaseWindow` as parameter.
          Defaults to `window.open()`.
-
         :param expected_class: Optional, check for the correct window class.
+        :param is_focused: Optional, waits until the new window will be focused.
+         Defaults to `True`.
```

Comment:
I think this parameter is a little confusing as documented. It doesn't just turn on the wait when is_focused=True, it waits for it not to be focused when is_focused=False. Would it be clearer if we just had "wait_for_focus" as the parameter? The case we're not expecting a new window to be focused seems more rare.